.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
This is a reminder to myself to add a `.nvm` file to use to fetch the Node version and use here to keep versions aligned. And alongside easing development and deployments 😅 .